docs/user_guide.md

```diff
@@ -556,76 +531,184 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
-![browse project](img/clair_not_ready.png)
+- For information about installing Harbor with Clair, see the [Installation and Configuration Guide](installation_guide.md). 
+- For information about adding additional scanners, see [Connect Harbor to Additional Vulnerability Scanners](#pluggable-scanners) below.
 
-The 'database not fully ready' warning message is also displayed in the **'Vulnerability'** tab of **'Configuration'** section under **'Administration'** for your awareness.
-![browse project](img/clair_not_ready2.png)
+You can manually initiate scanning on a particular image, or on all images in Harbor. Additionally, you can also set a policy to automatically scan all of the images at specific intervals.
 
-Once the database is ready, an overall database updated timestamp will be shown in the **'Vulnerability'** tab of **'Configuration'** section under **'Administration'**. 
+<a id="pluggable-scanners"></a>
+### Connect Harbor to Additional Vulnerability Scanners
+
+To connect Harbor to additional vulnerability scanners, you must have enabled the default Clair scanner when you deployed Harbor. You must install and configure an instance of the additional scanner according to the scanner vendor's requirements. The scanner must expose an API endpoint to Harbor that permits image pushes and pulls. You can deploy multiple different scanners, and multiple instances of the same type of scanner.
+
+1. Log in to the Harbor interface with an account that has Harbor administrator privileges.
+1. Expand **Administration**, and select **Interrogation Services**. 
+   ![Interrogation Services](img/interrogation-services.png)
+1. Click the **New Scanner** button.
+1. Enter the information to identify the scanner.
+   - A name for this scanner instance, to display in the Harbor interface.
+   - An optional description of this scanner instance.
+   - The address of the API endpoint that the scanner exposes to Harbor.
+   ![Add scanner](img/add-scanner.png)
+1. Select how to connect to the scanner from the **Authorization** drop-down menu.
+   ![Select scanner authentication method](img/scanner-auth.png)
+   - **None**: The scanner allows all connections without any security.
+   - **Basic**: Enter a username and password for an account that can connect to the scanner.
+   - **Bearer**: Paste the contents of a bearer token in the **Token** text box.
+   - **APIKey**: Paste the contents of an API key for the scanner in the **APIKey** text box.
+1. Optionally select **Skip certificate verification** if the scanner uses a self-signed or untrusted certificate. 
+1. Optionally select **Use internal registry address** if the scanner should connect to Harbor using an internal network address rather its external URL.
```

Comment:
>Optionally select Use internal registry address if the scanner should connect to Harbor using an internal network address rather than its external URL.

For this option, there must be a precondition, the scanner should be deployed in the same network in which scanner can reach Harbor with Harbor's internal network

Reply:
@steven-zou I added "**NOTE**: To use this option, the scanner must be deployed in a network that allows the scanner to reach Harbor via Harbor's internal network."